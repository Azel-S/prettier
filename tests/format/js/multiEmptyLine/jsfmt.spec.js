run_spec(__dirname, ["babel", "flow", "typescript"], {retainBlankLines: false});
run_spec(__dirname, ["babel", "flow", "typescript"], {retainBlankLines: true});