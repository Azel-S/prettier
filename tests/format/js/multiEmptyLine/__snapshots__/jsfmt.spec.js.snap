// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`multiEmptyLine.ts - {"retainBlankLines":false} format 1`] = `
====================================options=====================================
parsers: ["babel", "flow", "typescript"]
printWidth: 80
retainBlankLines: false
                                                                                | printWidth
=====================================input======================================
function add(x, y) {


    //comment
    

    return x + y;

    //comment



  }
=====================================output=====================================
function add(x, y) {
  //comment

  return x + y;

  //comment
}

================================================================================
`;

exports[`multiEmptyLine.ts - {"retainBlankLines":true} format 1`] = `
====================================options=====================================
parsers: ["babel", "flow", "typescript"]
printWidth: 80
retainBlankLines: true
                                                                                | printWidth
=====================================input======================================
function add(x, y) {


    //comment
    

    return x + y;

    //comment



  }
=====================================output=====================================
function add(x, y) {
  //comment


  return x + y;

  //comment
}

================================================================================
`;
